script/Arcueid/Day 7/07_00_ARC07_3F.txt

```diff
@@ -34,7 +34,7 @@ Yet haven't you also told me lies?"
 -- 「む。なにその言い方。すっごくいやらしいんですけど」
 // Skihi VA is somewhat accusing, sulky so 言い方 -> tone
 "Hmph. What's with that tone?
- You're being extremely unpleasant you know."
+ You're being extremely unpleasant, you know."
```

Comment:
kinda think revert this one. Grammatically correct or no, I think it reads cleaner without the comma

Reply:
Interesting, I was actually pro- this one. Comma placement is heavily subjective though :aokoheh: